app/src/main/java/com/onboarding/pronosticAR/utils/DateUtils.kt

```diff
@@ -0,0 +1,23 @@
+package com.onboarding.pronosticAR.utils
+
+object DateUtils {
+    fun getDate(date: String) =
+        date.substring(STRING_DAY_START, STRING_DAY_END) +
+        SLASH +
+        date.substring(STRING_MONTH_START, STRING_MONTH_END) +
+        SPACE +
+        date.substring(STRING_HOUR_START, STRING_HOUR_END) +
+        COLON +
+        STRING_MINUTES
```

Comment:
here you can use String.format() 
for example
val pi = 3.14159265358979323
val formatted = String.format("%.2f", pi)

Reply:
kind of done